docs/clusters/alpine/condo-fairshare-and-resource-access.md

```diff
@@ -10,7 +10,7 @@ Updated November 4, 2022
 
 The Alpine supercomputer hosted at CU Boulder and administered by Research 
 Computing at CU Boulder (CURC) supports institutional condo purchases, the 
-first two of which come online in the fall of 2022. This document outlines 
+first two of which came online in the fall of 2022. This document outlines 
```

Comment:
On this page there is a "Updated November 4, 2022" part at the top. I think we should remove this.